modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountGroupResourceImpl.java

```diff
@@ -222,7 +222,8 @@ public Response patchAccountGroupByExternalReferenceCode(
 
 		if ((customFields != null) && !customFields.isEmpty()) {
 			ExpandoUtil.updateExpando(
-				contextCompany.getCompanyId(), CommerceAccountGroup.class,
+				contextCompany.getCompanyId(),
+				com.liferay.account.model.AccountGroup.class,
```

Comment:
Why do you need to write the whole package and not use just the class name?
`AccountGroup.class`

Reply:
AccountGroup is the DTO used in the headless, so we have to use the whole package to make sure we are using the proper class